### ISSUE TITLE
Auto Pick Causing Quest to Not be able to be completed

### ISSUE THREAD

**Issue by TLJQ** [Gktwo/wuwa-mod#32]
The auto pick feature has caused my quest "The Past is the Future: Continued - Open the object left by Shiyu" to be bugged.
Seems like during the dialogue the auto pick feature had opened the object, and now there is nothing at the waypoint where I am supposed to open the object.

**Comment by Poliku10** [comment]
yea me too and I can't finish the quest because I don't have anything to open. Guess we need to contact cs.

**Comment by SUCAMADIC** [comment]
yeah pretty much every auto loot will not work great in this game because there is tidal chest where you need to defeat monster to open so not really recommended if you wanna 100%
